lib/src/play_store_search_api.dart

```diff
@@ -59,7 +59,7 @@ class PlayStoreResults {
     try {
       final sectionElements = response.getElementsByClassName('W4P4ne');
       final releaseNotesElement = sectionElements.firstWhere(
-        (elm) => elm.querySelector('.wSaTQd')!.text == 'What\'s New',
+        (elm) => elm.querySelector('.wSaTQd')!.text == 'What\'s New',orElse:()=> sectionElements[0]
```

Comment:
@yassinsameh This change failed the formatter test. Can you fix that?
```
$ flutter format -n --set-exit-if-changed ./lib ./test ./example || travis_terminate 1
Changed lib/src/play_store_search_api.dart
Formatted 24 files (1 changed) in 0.66 seconds.
Formatting failed: 1
```

Reply:
Hi @larryaasen, Apologies for the late response. I'm currently travelling, i have no idea why it would fail the formatter test, sorry. Will give an update as soon as i can if you haven't found a solution. Thank you.